app/controllers/users_controller.rb

```diff
@@ -4,6 +4,7 @@ def index
   end
 
   def show
-    @user_id = params[:id]
+    @user = User.find(params[:id])
+    @recent_posts = @user.recent_posts
   end
 end
```

Comment:
- [Optional] Dear @dialite, I'm not sure if it's required to implement the `users#index` action in the previous milestone. But if it was not, I would suggest including all the changes in the new PR. This help who ever reviews your project to see all your changes. 
> As you can see the `users#index` action is not visible in the file changed tab, but I can see it existed once I closed the project.

Reply:
Okay well noted.